### PR TITLE
Fixed Registration not working issue

### PR DESCRIPTION
**Pull request by lemohammed**
Changelog:
- ../components/RegisterFormComp.tsx
- Added first and last name states and added new input for last name.
- Fixed user class static method call

### CODE REVIEW

**Review by RatikKapoor — APPROVED** [review]
🚀

File: src/components/RegisterFormComp.tsx [hunk 1]
```diff
@@ -33,7 +38,12 @@ const RegisterForm: React.FC = () => {
             <IonInput
                 className="registerInput"
                 placeholder="Username"
-                onIonChange={(e: CustomEvent<InputChangeEventDetail>) => setUsername(e.detail.value ?? '')}
+                onIonChange={(e: CustomEvent<InputChangeEventDetail>) => setFirstName(e.detail.value ?? '')}
+            />
+            <IonInput
```
Comment (RatikKapoor) [inline src/components/RegisterFormComp.tsx:43]: This `IonInput` is a duplicate it looks like
image

**Review by RatikKapoor — CHANGES_REQUESTED** [review]

File: src/components/RegisterFormComp.tsx (hunk 1 above)
Comment (lemohammed) [reply src/components/RegisterFormComp.tsx:43]: Oh that place holder should say first then last name

Comment (RatikKapoor) [reply src/components/RegisterFormComp.tsx:43]: Gotcha! I'll go ahead and approve the PR as soon as that's fixed!

**Review by RatikKapoor — APPROVED** [review]
💯

**Review by MrClean-1 — APPROVED** [review]

**Pull request merged**